test/cmd/annotations.sh

```diff
@@ -12,17 +12,17 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/annotate"
 # This test validates empty values in key-value pairs set by the annotate command
-os::cmd::expect_success_and_text 'oc create -f examples/hello-openshift/hello-pod.json' 'pod "hello-openshift" created'
-os::cmd::expect_success_and_text 'oc annotate pod hello-openshift node-selector=""' 'pod "hello-openshift" annotated'
-os::cmd::expect_success_and_not_text 'oc get pod hello-openshift --template="{{index .metadata.annotations \"node-selector\"}}"' '.'
+os::cmd::expect_success_and_text 'oc create -f examples/hello-openshift/hello-pod.json' 'pod/hello-openshift created'
+os::cmd::expect_success_and_text 'oc annotate pod hello-openshift node-selector=""' 'pod/hello-openshift annotated'
+os::cmd::expect_success_and_not_text 'oc get pod hello-openshift --output=go-template="{{index .metadata.annotations \"node-selector\"}}"' '.'
```

Comment:
This one seems bad.  Why did the old call start to fail?

Reply:
The old call fails because it did not provide an --output argument.
Per the changes introduced when we wired up the template PrintFlags, we no longer implicitly assumed that a template output-format was meant whenever the `--template` flag was provided [1].

As of [1], 
- we can now provide the template argument via `--template`, but still have to specify which output format we want (jsonpath, go-template, etc.)
- we can omit the `--template` flag entirely and specify both the output format and the template argument as the value of `--output` (this was already possible before [1]).

1. https://github.com/kubernetes/kubernetes/pull/61304/files#diff-6ee7dad6fdeccf4e8fe8669b7080b962R70